src/core/model_config_utils.cc

```diff
@@ -492,6 +492,20 @@ ValidateModelConfig(
     }
   }
 
+  Status status;
```

Comment:
I think this is a good change, but should be moved to line 360, and the input/output checks there should be moved into ValidateModelInputOutput (likely need to pass in max_batch_size).

The history is that input/output validation used to be in here but I forget why it moved out and probably this is now better. But you need to check where the current usage is in the bundles and remove the direct calls to ValidateModelInput/Output. Also I recently added explicit calls to these in the unit-test harness and those can now be removed.